tests/System.Text.Primitives.Tests/Encoding/Utf8UtilTests.Validation.cs

```diff
@@ -274,7 +274,7 @@ private static void AssertIsInvalidTwoByteSequence(byte[] invalidSequence)
         {
             Assert.Equal(2, invalidSequence.Length);
 
-            byte[] KNOWN_GOOD_BYTES = DecodeHex(E_ACUTE);
+            byte[] KNOWN_GOOD_BYTES = TestHelper.DecodeHex(E_ACUTE);
```

Comment:
nit: use camel casing

Reply:
Existing unit tests are out of scope of this review. I'll handle it in a separate PR.